types/errors.go

```diff
@@ -80,7 +80,7 @@ func HandleServerError(writer http.ResponseWriter, err error) {
 	case *ItemNotFoundError:
 		respErr = responses.SendNotFound(writer, err.Error())
 	case *AuthenticationError:
-		respErr = responses.SendForbidden(writer, err.Error())
+		respErr = responses.SendUnauthorized(writer, err.Error())
```

Comment:
I dont think `AuthenticationError` always means Unauthorized. 
Forbidden is a valid response for some places where `AuthenticationError` is used

Reply:
As far as I can see, we use it only when reporting problems with authentication, we can add another type of error tho

Reply:
OK. Also are we sure that OCM or anything else isn't relying on the response codes? Just checking

Reply:
What do you think about the new changes?

Reply:
makes more sense to me, thx!